src/modules/Channel/context/dux/actionTypes.js

```diff
@@ -2,6 +2,8 @@ export const RESET_MESSAGES = 'RESET_MESSAGES';
 
 export const FETCH_INITIAL_MESSAGES_START = 'FETCH_INITIAL_MESSAGES_START';
 export const FETCH_INITIAL_MESSAGES_SUCCESS = 'FETCH_INITIAL_MESSAGES_SUCCESS';
+export const INITIAL_MESSAGES_IS_SCROLLED = 'INITIAL_MESSAGES_IS_SCROLLED';
```

Comment:
Not sure it's good to manage the scroll state with dux since it's not really related to our business logic(generally we manage business related state with dux). Why don't we control this flag with ChannelContext?